index.html

```diff
@@ -77,26 +77,27 @@ <h2>
                 </section>
 
                 <!-- Projects -->
-                <section class="projects" id="projects">
+                <section class="projects__wrapper" id="projects">
                     <h2>
-                        <span class="overheading">What Did I Do</span>
+                        <span class="overheading overheading--light-blue">What Did I Do</span>
                         Projects
                     </h2>
 
                     <!-- Project Alfa -->
-                    <article>
+                    <article class="article">
```

Comment:
má být součástí komponenty projects
alternativně by mohla být nová komponenta `project`
```suggestion
                    <article class="projects__project">
```